sqlalchemy_redshift/dialect.py

```diff
@@ -301,7 +304,13 @@ def _fetch_redshift_column_attributes(self, column):
         return text
 
 
-class RedshiftDialect(PGDialect_psycopg2):
+class RedshiftExecutionContext(pgdialect.PGExecutionContext_psycopg2):
+    def should_autocommit_text(self, statement):
+        return super(RedshiftExecutionContext, self).should_autocommit_text() \
+            or AUTOCOMMIT_REGEXP.match(statement)
```

Comment:
I generally prefer to use parentheses for line continuations rather than `\`:
```
        return (super(RedshiftExecutionContext, self).should_autocommit_text()
                    or AUTOCOMMIT_REGEXP.match(statement))
 
```